meson.build

```diff
@@ -69,7 +71,15 @@ if enable_breadcrumbs
   add_project_arguments('-DVKD3D_ENABLE_BREADCRUMBS', language : 'c')
 endif
 
-vkd3d_external_includes = [ './subprojects/Vulkan-Headers/include', './subprojects/SPIRV-Headers/include' ]
+if enable_tracy
+  add_project_arguments('-DTRACY_ENABLE', language : 'c')
+
+  if dynamic_tracy
+    add_project_arguments('-DDYNAMIC_TRACY', language : 'c')
+  endif
+endif
+
+vkd3d_external_includes = [ '../../common/vulkan/include', './subprojects/SPIRV-Headers/include' ]
```

Comment:
`common/vulkan/include` what is that?

Reply:
Code mistakenly added from Juice's private builds.